apps/taquito-test-dapp/src/tests.ts

```diff
@@ -316,6 +316,30 @@ const signPayloadAndSend = async (
   }
 };
 
+const signFailingNoop = async (
+  input: string,
+  tezos: TezosToolkit,
+  wallet: BeaconWallet,
+): Promise<TestResult> => {
+  const bytes = char2Bytes(input);
+  try {
+    const signedPayload = await tezos.wallet.signFailingNoop({
+      arbitrary: bytes,
+      basedOnBlock: 'head'
+    });
+
+    return {
+      success: true,
+      opHash: "",
+      output: signedPayload.prefixSig,
+      sigDetails: { input, bytes: signedPayload.bytes, formattedInput: input }
```

Comment:
might be a good idea to replace formattedInput to `{ signedContent: {
      branch: `blockhash...`,
      contents: [{kind: OpKind.FAILING_NOOP, arbitrary: `123...`}]
    }`